corehq/apps/app_manager/suite_xml/post_process/workflow.py

```diff
@@ -95,14 +95,8 @@ def get_frame_children(self, command, module, module_only=False, include_root_mo
         :returns:   list of strings and DatumMeta objects. String represent stack commands
                     and DatumMeta's represent stack datums.
         """
-        match = re.search(r'^(m\d+)-(.*)', command)
-        if not match:
-            raise Exception("Unrecognized command: {}".format(command))
-        module_id = match.group(1)
-        form_id = match.group(2)
         module_command = id_strings.menu_id(module)
-        module_datums = self.get_module_datums(module_id)
-        form_datums = module_datums[form_id]
+        module_datums = self.get_module_datums(f'm{module.id}')
```

Comment:
I don't love adding `m` to the beginning here - it'd be nice to make this whole file use _just_ the id for each of these things.  I tried that, but kept getting issues and ended up punting.